src/Mvc/Mvc.Razor.RuntimeCompilation/src/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj

```diff
@@ -23,6 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="build\$(DefaultNetCoreTargetFramework)\*" Pack="true" PackagePath="build\$(DefaultNetCoreTargetFramework)" />
+    <None Include="targets\$(PackageId).targets" Pack="true" PackagePath="build\$(DefaultNetCoreTargetFramework)\$(PackageId).targets" />
```

Comment:
Who sets packageid? Also, are we renaming the folder name to netcoreapp3.1?

Reply:
PackageId is set by the SDK. So this should result in

`<None Include="targets\Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.targets" Pack="true" PackagePath="build\netcoreapp3.1\Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.targets" />`